### DISCUSSION TITLE
auto reconnect

### DISCUSSION THREAD

**Discussion by tk746** [PrismarineJS/mineflayer#1972] [Q&A] [answered]
is there a way to make my bot reconnect if it were to get kicked or lagged out? automatically

**Comment by Hazuki-san** [comment]
example should have it

**Comment by amoraschi** [comment, marked as answer]
This question has been asked so many times. You just need to search it through Discussions or Discord

**Reply by DEATHToHacker** [reply]
can you give me a link for discord?

**Reply by imharvol** [reply]
It's on the README.md: https://discord.gg/GsEFRM8

**Comment by TheoPerson** [comment]
> is there a way to make my bot reconnect if it were to get kicked or lagged out? automatically

Hi, do you get any serious answers or code for your issue? I have the same issue and I'm searching for the solution ^^

**Reply by u9g** [reply]
look at the reconnector example

**Reply by tk746** [reply]
i can help you i got it now :)
